DOCS-6249 rename (#121)
mulesoft/docs-api-community-manager: modules/ROOT/pages/configure-acm.adoc

== Create and Authenticate the External Data Source [[create-and-authenticate-the-external-data-source]]

> The external data source specifies how API Community Manager components and data objects interact securely with your MuleSoft Anypoint organization.

Reviewer: MuleSoft Anypoint organization > Anypoint Platform organization?

Author: Yes, that is better. Thank you!

*Password* fields, respectively, for a system user with read-only privileges in your Anypoint Platform organization.
. Click *Save*.

== Enable API Community Manager Control Panel [[enable-acm-control-panel]]

> The API Community Manager control panel is the interface your community administrators and operators will use to operate your communities. It is available in the *App Launcher* where it is the first icon displayed in the main navigation bar. To ensure that it is available for admin users:

Reviewer: ```suggestion
The API Community Manager control panel is the interface your community administrators and operators use to operate your communities. It is available in the *App Launcher* page, where it is the first icon displayed in the main navigation bar. To ensure that it is available for admin users:
```

. In *Setup*, search for *App Manager* using the *Quick Find* box and click *App Manager*.
. Locate the *API Community Manager* item, and in the menu on the right end of the row, click *Edit*.
. Click *Navigation Items* on the left panel, highlight the *ACM Administrator* item under the *Available Items* box, and use the arrow buttons to move it to the *Selected Items* box. Click *Save*.
. Click *Back* at the top right.
. Log out and log back in to your Salesforce organization.
. Open the *App Launcher* and click *API Community Manager*.

> . Verify that the API Community Manager control panel is displayed correctly. In the *Application Launcher*, open *ACM Administrator*. If you see a message that you have not created a community, the actions in this step were done correctly.

Reviewer: GLOBALLY ensure that steps and list items do not exceed one sentence, and that that sentence includes the specific user action:
```suggestion
. Verify that the API Community Manager control panel is displayed correctly by navigating to *Application Launcher* > *ACM Administrator*. 
If you see a message that you have not created a community, then the actions in this step were performed correctly.
```

. Click *Save* so this setting is reflected.
. Click *Allow discussion threads*.
. Click *Save*.

> . Go back to the API Community Manager control panel, and click *Manage CMS Content*.

Reviewer: ```suggestion
. Return to the API Community Manager control panel, and click *Manage CMS Content*.
``

By default discussion forums are not moderated, so any member can create posts with any content and the posts become visible for other members immediately. If you want to define a basic "review/approve-all" moderation model, use these steps.

> . In the API Community Manager control panel, click *Manage Forums* and click *Rules*.

Reviewer: ```suggestion
. In the API Community Manager control panel, click *Manage Forums* > *Rules*.
```

=== Publish Your Community [[publish-your-community]]

. In the API Community Manager control panel, click *Community Builder*.
. Click *Publish* on the top right corner, and click *Publish* in the *Publish Your Community* dialog.

> . In a few minutes your community will be published, and you will receive an email notification including your community's public URL. You can also find your community live URL by opening the API Community Manager control panel and clicking *Open Community*.

Reviewer: This is not a step, although you might reword it as one:
". In a few minutes, look for an email notification...URL.
You can also...."

Author: Fixed-- thank you!

=== Configure Guest User Access [[configure-guest-user-access]]

> . In the API Community Manager control panel, open *Community Builder*, and click *Settings* -> *General*.

Reviewer: ```suggestion
. In the API Community Manager control panel, open *Community Builder* and click *Settings* > *General*.
```

=== Enable Guest User Registration as Members [[enable-guest-user-registration-as-members]]

These steps enable guests to register as members in your API Community Manager portal.

==== Add Self Register Component to Registration Page [[add-self-register-custom-component-to-register-page]]

> The self-registration Lightning component enables guest users to register themselves as members of your community. If you're not using any of the API Community Manager out-of-the-box templates, use these steps to add the self-registration Lightning component specific to API Community Manager.

Reviewer: Oh the debates we have had over "out-of-box" and "out-of-the-box"!
Consider "If you're not using any of the templates supplied with API Community Manager, follow these steps to...."

Author: That's much better! Fixed.

. In the API Community Manager control panel, open *Community Builder* and navigate to the *Register* page in the Pages menu on the top left.
. Remove the standard *Self Registration* Lightning component by clicking the delete icon next to it.
. Navigate to *Components*, search for *Self Register* and add the Lightning component to the page.

==== Assign a Profile for Your Community Members [[assign-a-profile-for-your-community-members]]

> . In the API Community Manager control panel, open *Community Administration* and navigate to the *Members* section on the left panel.

Reviewer: on > in

Author: Fixed.

NOTE: Refer to xref:brand-intro.adoc[Brand and Customize Your API Community] to understand how to configure the look and feel of your login and registration pages.

> . In the API Community Manager control panel, open *Community Administration* and navigate to the *Login & Registration* section on the left panel.

Reviewer: ditto:
```suggestion
. In the API Community Manager control panel, open *Community Administration* and navigate to the *Login & Registration* section in the left panel.
```

===== Activate the Approval Process [[activate-the-approval-process]]

. On the *Approve Registration* page, click *Activate* and confirm.

> . In the API Community Manager control panel, open *Community Builder*, and click *Settings* -> *General*.

Reviewer: ```suggestion
. In the API Community Manager control panel, open *Community Builder* and click *Settings* > *General*.
```

> . In Community Builder, navigate to the *Register* page in the pages menu and select the API Community Manager *Self Register* Lightning component.

Reviewer: ```suggestion
. In Community Builder, navigate to the *Register* page in the Pages menu and select the API Community Manager *Self Register* Lightning component.
```

.. Click the plus button.
. To apply these changes, log out and then log back in.

> . Go to *Secrets Manager* and select the `ACM` environment using the environment switcher. Click *Create Secret Group* and enter this information.

Reviewer: Separate into two steps:
```suggestion
. Go to *Secrets Manager* and select the `ACM` environment using the environment switcher. 
. Click *Create Secret Group* and enter this information.
```
